src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -332,4 +299,211 @@ private function updateGitLabProject(array $project): void
             }
         }
     }
+
+    private function getCloudCodebase(string $codebaseUuid): CodebaseResponse
+    {
+        $codebasesResource = new Codebases($this->cloudApiClientService->getClient());
+        return $codebasesResource->get($codebaseUuid);
+    }
+
+    private function determineCloudCodebase(): ?string
+    {
+        $codebaseUuid = $this->doDetermineCloudCodebase();
+        if (!isset($codebaseUuid)) {
+            throw new AcquiaCliException("Could not determine Cloud Codebase");
+        }
+
+        return $codebaseUuid;
+    }
+
+    /**
+     * @throws \Acquia\Cli\Exception\AcquiaCliException
+     */
+    private function doDetermineCloudCodebase(): ?string
+    {
+        $acquiaCloudClient = $this->cloudApiClientService->getClient();
+
+        if ($this->input->isInteractive()) {
+            /** @var CodebaseResponse $codebase */
+            $codebase = $this->promptChooseCodebase($acquiaCloudClient);
+            if ($codebase) {
+                return $codebase->id;
+            }
+        }
+
+        return null;
+    }
+
+    /**
+     * Prompts the user to choose from a list of available Cloud Platform
+     * codebases.
+     *
+     * @throws \Acquia\Cli\Exception\AcquiaCliException
+     */
+    private function promptChooseCodebase(
+        Client $acquiaCloudClient
+    ): object|array|null {
+        $codebasesResource = new Codebases($acquiaCloudClient);
```

Comment:
There's going to be overlap with @jigar-shah-acquia work in https://github.com/acquia/cli/pull/1877, please make sure to coordinate so as not to duplicate functionality